### PR TITLE
rectified the helm commands for AKS

### PR DESCRIPTION
**Pull request by samirparhi-dev**
**Description**
rectified the helm commands for AKS

**Notes for Reviewers**
rectified the helm commands for AKS

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8b323487d8efb75b4dd8db65d5b18d233b8b9d01 at: https://619f84f56d294a2ea0afe176--meshery-docs-preview.netlify.app

File: docs/pages/installation/aks.md [hunk 1]
```diff
@@ -61,14 +63,14 @@ kubectl cluster-info
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
  $ kubectl create ns meshery
- $ helm repo add meshery https://meshery.io/charts/
- $ helm install meshery --namespace meshery
+ $ helm repo add meshery https://meshery.io/charts
+ $ helm install meshery meshery/meshery --namespace meshery
```
Comment (Aisuko) [inline docs/pages/installation/aks.md:67]: Where is that path `meshery/meshery` ?

Comment (Aisuko) [reply docs/pages/installation/aks.md:67]: If we add the meshery Helm chart's repo, we can install meshery directly right? Why we need to add the path of the chart?

**Review by Aisuko — COMMENTED** [review]
Thanks for contributing, please take a look the comment below.

File: docs/pages/installation/aks.md (hunk 1 above)
Comment (panyuenlau) [reply docs/pages/installation/aks.md:67]: @Aisuko the first `meshery` of `meshery/meshery` refers to the added repo name, and the second `meshery` refers to the chart name under the repo.

The commands look good to me, except I'd prefer to combine `kubectl create ns meshery` and `helm install` into one command by adding the `--create-namespace` flag, e.g:
`helm install meshery meshery/meshery --namespace meshery --create-namespace`

Comment (Aisuko) [reply docs/pages/installation/aks.md:67]: Ok, thanks @panyuenlau. For me, I'd like to suggest add the comment to the command above. And `--create-namespace` is a great parameter for deploying the Helm chart without create namesapce manually.

**Comment by Aisuko** [comment]
The PR move to #4781 due to fix the conflicts.

**Pull request closed without merging**